### PR TITLE
Remove issue & PR templates

### PR DESCRIPTION
**Pull request by jennuine**
Using [default community templates](https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file) from https://github.com/ignitionrobotics/.github 

This provides a single set of templates across our entire organization.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -46,12 +46,12 @@ context (e.g., screenshots, gifs) if appropriate.-->
...
-to support the maintainers
+- [ ] While waiting for a review on your PR, please help review
+[another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers
 
 **Note to maintainers**: Remember to use **Squash-Merge**
 
----
+🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸
 
 # Forward port
```
Comment (chapulina) [inline .github/PULL_REQUEST_TEMPLATE.md:56]: How about :arrow_right::

```suggestion
# ➡️ Forward port
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Branch comparison: https://github.com/ignitionrobotics/ign-cmake/compare/<TO_BRA
 
 **Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)
 
----
+🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸
 
 <!-- For maintainers only -->
```
Comment (chapulina) [inline .github/PULL_REQUEST_TEMPLATE.md:67]: For the release, how about :arrow_up: ?

**Review by chapulina — APPROVED** [review]
Nice! :sparkles: I have a couple of suggestions if we want emojis for all sections :sweat_smile:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jennuine) [reply .github/PULL_REQUEST_TEMPLATE.md]: :heavy_check_mark: 4865cb95b60742e8542ab1f352f1a187ddb8d7c5

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (jennuine) [reply .github/PULL_REQUEST_TEMPLATE.md:67]: :heavy_check_mark: but with :balloon: 4865cb95b60742e8542ab1f352f1a187ddb8d7c5

**Review by chapulina — APPROVED** [review]

**Pull request merged**